app/services/flow/flows/provider_capital.rb

```diff
@@ -34,30 +42,36 @@ class ProviderCapital < FlowSteps
           check_answers: :means_summaries,
         },
         identify_types_of_outgoings: {
-          path: ->(application) { urls.providers_legal_aid_application_identify_types_of_outgoing_path(application) },
+          path: ->(application) { urls.providers_legal_aid_application_means_identify_types_of_outgoing_path(application) },
           forward: lambda do |application|
-            if application.uploading_bank_statements?
-              application.transaction_types.debits.any? ? :cash_outgoings : :has_dependants
+            if application.outgoing_types?
+              :cash_outgoings
+            elsif application.income_types?
+              :income_summary
             else
-              :outgoings_summary
+              :has_dependants
             end
           end,
           check_answers: lambda do |application|
-            if application.uploading_bank_statements?
-              application.transaction_types.debits.any? ? :cash_outgoings : :means_summaries
-            else
-              :outgoings_summary
-            end
+            application.outgoing_types? ? :cash_outgoings : :means_summaries # could this intend to point to outgoings_summary? need to check CYA behaviour
           end,
         },
         cash_outgoings: {
           path: ->(application) { urls.providers_legal_aid_application_means_cash_outgoing_path(application) },
-          forward: :has_dependants,
+          forward: lambda do |application|
+            if application.income_types?
+              :income_summary
+            elsif application.outgoing_types?
+              :outgoings_summary
+            else
+              :has_dependants
+            end
+          end,
           check_answers: :means_summaries,
         },
-        # Dependant steps here (see ProviderDependants)
-        # Property steps here (see ProviderProperty)
-        # Vehicle steps here (see ProviderVehicle)
+        # this page is not mentioned in the flow but we have a required question on it and this is the only
+        # place we replay the accounts back to the provider. I have added it here because it is just before we ask users
+        # to organise the transactions from these accounts.
```

Comment:
I will remove this comment as the page has been moved back to it's previous location